azure-pipelines.yml

```diff
@@ -29,14 +29,16 @@ jobs:
         steps:
         - bash: |
             echo "y" | $ANDROID_HOME/tools/bin/sdkmanager --install 'system-images;android-${{ value }};google_apis;x86'
-            # -d 9 for Nexus 5X
+            # -d 17 for Pixel
             # to find out other devices run "avdmanager list device"
-            echo "no" | $ANDROID_HOME/tools/bin/avdmanager create avd -n xamarin_android_emulator -k 'system-images;android-${{ value }};google_apis;x86' -d 9 --force
+            echo "no" | $ANDROID_HOME/tools/bin/avdmanager create avd -n xamarin_android_emulator -k 'system-images;android-${{ value }};google_apis;x86' -d 17 --force
 
             $ANDROID_HOME/emulator/emulator -list-avds
 
             echo "Starting emulator"
-            nohup $ANDROID_HOME/emulator/emulator -avd xamarin_android_emulator -skin 1080x1920 -no-snapshot > /dev/null 2>&1 &
+            # increase memory size to 2048M to avoid error "System UI is not responding"
+            sed -i 's/^\(hw.ramSize=\).*$/\12048M/' ~/.android/avd/xamarin_android_emulator.avd/config.ini
```

Comment:
What does \1 mean?

Reply:
And also please add comment about it

Reply:
It means to keep the part that is in braces in the left-side expression. So I will get `hw.ramSize=96M` -> `hw.ramSize=2048M`. Without it I will get `hw.ramSize=96M` -> `2048M`